packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -68,6 +78,8 @@ class Autocomplete<T extends Object> extends StatelessWidget {
     this.optionsViewBuilder,
     this.optionsViewOpenDirection = OptionsViewOpenDirection.down,
     this.initialValue,
+    this.showOptionsViewOnEmptyOptions = false,
+    this.showOptionsViewOnUncompletedOptions = false,
```

Comment:
Maybe this instead of "uncompleted": showOptionsViewOnUncompletedOptions => showOptionsViewOnPendingOptions